index.html

```diff
@@ -516,20 +516,12 @@ <h2 id="3Definitions">Definitions</h2>
 
 <dd>the red, green, blue and alpha components are all set to zero.</dd>
 
+<dt><dfn>gamma value</dfn></dt>
+<dd>value of the exponent of a <a>gamma</a> <a>transfer function</a></dd>
+
 <!-- Maintain a fragment named "3gamma" to preserve incoming links to it -->
 <dt id="3gamma"><dfn>gamma</dfn></dt>
-
-<dd>exponent that describes approximations to certain non-linear
-transfer functions encountered in image capture and reproduction.
-Within this specification, gamma is the exponent in the
-transfer function from <tt>display_output</tt> to
-<tt>image_sample</tt>
-<pre>
-<tt>image_sample = display_output<sup>gamma</sup></tt>
-</pre>
-where both <tt>display_output</tt> and <tt>image_sample</tt>
-are scaled to the range 0 to 1.
```

Comment:
Seems important to still point out that component values are in the range [0..1] when the power law is applied.(Not [0..255] for example).

Reply:
This is detailed elsewhere in the document, e.g. at 13.12.

Also, a definition should be written such that it can replace the term in its context.